src/openforms/submissions/api/serializers.py

```diff
@@ -221,6 +224,8 @@ class SubmissionStepSerializer(NestedHyperlinkedModelSerializer):
         "submission_uuid": "submission__uuid",
     }
 
+    instance: SubmissionStep
```

Comment:
`class SubmissionStepSerializer(NestedHyperlinkedModelSerializer[SubmissionStep]):` can be used to infer the `instance` type as well (but it seems to be either `SubmissionStep | Sequence[SubmissionStep]` from stubs so not really useful)